<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ACInjectorUI</class>
 <widget class="QMainWindow" name="ACInjectorUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>805</width>
    <height>398</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ACInjectorUI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <widget class="QLabel" name="ACLogo">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../Images/ACEZModeLogo.png</pixmap>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="ProcessFoundIcon">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>140</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../Images/checkmark-512.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="ProcessFoundText">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>140</y>
      <width>591</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Assault Cube Process Detected</string>
    </property>
   </widget>
   <widget class="QLabel" name="InjectableFoundIcon">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>180</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../Images/checkmark-512.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="InjectableFoundText">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>180</y>
      <width>601</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Injectable DLL Found</string>
    </property>
   </widget>
   <widget class="QLabel" name="LocationLabel">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>220</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Location: </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LocationTextField">
    <property name="geometry">
     <rect>
      <x>182</x>
      <y>220</y>
      <width>411</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="LocationBrowseButton">
    <property name="geometry">
     <rect>
      <x>599</x>
      <y>214</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QPushButton" name="InjectEjectButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>280</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Inject</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CheckProcessButton">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>140</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Check Again</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>805</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
